pom.xml

```diff
@@ -99,6 +99,7 @@
 		<dependency>
 			<groupId>software.amazon.awssdk</groupId>
 			<artifactId>apigateway</artifactId>
+			<version>2.21.5</version>
```

Comment:
I think you can pass the `aws.java.sdk.version` variable into here instead of duplicating version numbers across dependencies.

Reply:
Good shout, will change this.

Reply:
in this case you don't need to specify the version, as in dependencyManagement(line 23) will take care of all aws dependency

Reply:
in this case you don't need to specify the version, as in dependencyManagement(line 23) the amazone bom will take care of all aws dependency

Reply:
Nice, I'll just remove that line 👍